plugins/ccxtExchange_test.go

```diff
@@ -587,28 +581,20 @@ func TestGetOrderConstraints_Ccxt_Precision(t *testing.T) {
 		},
 	}
 
-	for _, kase := range testCases {
-		t.Run(kase.exchangeName, func(t *testing.T) {
-			testCcxtExchange, e := makeCcxtExchange(
-				kase.exchangeName,
-				nil,
-				[]api.ExchangeAPIKey{emptyAPIKey},
-				[]api.ExchangeParam{emptyParams},
-				[]api.ExchangeHeader{},
-				false,
-				getEsParamFactory(kase.exchangeName),
-			)
-			if !assert.NoError(t, e) {
-				return
-			}
+	for _, tc := range testCases {
```

Comment:
Can we leave the var as kase to keep consistent with rest of codebase?
Tc is probably a better abbreviation/var-name but can migrate to that later in one sweeping change